### ISSUE TITLE
MPAdViewDelegate do not provide an error

### ISSUE THREAD

**Issue by numen31337** [mopub/mopub-ios-sdk#264]
One line summary of the issue here

- [x] I am submitting a bug report for existing functionality
- [x] I visited https://developers.mopub.com/ and found no answer
- [x] I checked https://twittercommunity.com/c/advertiser-api/mopub to make sure that this issue has not already been filed
- [x] I checked to make sure that this issue has not already been filed

#### MoPub SDK Version:
5.6.0

#### Expected behavior:
```objective-c
- (void)adViewDidFailToLoadAd:(MPAdView *)view;
```
`adViewDidFailToLoadAd` should provide the developers with the error information which can be dug up down in the call stack here where this method is being called:

```objective-c
- (void)didFailToLoadAdapterWithError:(NSError *)error
{
    [self.delegate managerDidFailToLoadAd]; //<--- Error is skiped here
    [self scheduleRefreshTimer];
}
```
#### Observed behavior:
No error information provided.

**Comment by dravyan** [comment]
Hi @numen31337,
 Thank you for reporting this. We appreciate the feedback on having error information on `adViewDidFailToLoadAd`. We have worked on the fix and will be available in our next SDK release. Please look out for SDK release notes.

Thanks,
Dravya Nataraj
MoPub SDKTeam

**Comment by dravyan** [comment]
This issue has been fixed in SDK version 5.7.0. Thank you.

-Dravya Nataraj
MoPub SDK Team
